en/scripting/language-support.md

```diff
@@ -0,0 +1,89 @@
+# Language Support
+
+## JavaScript
```

Comment:
Please remove the whole JavaScript section, we no longer support create JavaScript files in our editor. The only script option is TypeScript

Reply:
ok